### PR TITLE
Add V4l2 test plan

### PR DESCRIPTION
**Pull request by ana**
This is an initial test plan that runs v4l2-compliance in the system and parses the output.
Example: https://lava.collabora.co.uk/results/1167325/0_v4l2

### CODE REVIEW

File: lava-v2-jobs-from-api.py [hunk 1]
```diff
@@ -45,6 +45,8 @@
 NFSROOTFS_URL = '/'.join([ROOTFS_URL, '{}', 'rootfs.tar.xz'])
 KSELFTEST_INITRD_URL = '/'.join([ROOTFS_URL, '{}', 'tests', 'rootfs.cpio.gz'])
 
+TESTS_ROOTFS_URL = 'http://staging-storage.kernelci.org/images/rootfs/debian/stretchtests/20180601.3/'
```
Comment (gctucker) [inline lava-v2-jobs-from-api.py]: This should be on `storage.kernelci.org`, not `staging-storage`.

Is the job to build and publish this file system all merged and deployed now?

Comment (ana) [reply lava-v2-jobs-from-api.py]: No, the jobs to build and publish debian images are not still deployed in api/storage.kci.org
This is blocked by https://github.com/kernelci/kernelci-build-staging/pull/35
I'm waiting for builder10 to be back to test a PR for that.

**Comment by gctucker** [comment]
@ana Do you have a LAVA job running this test with your latest patches (production rootfs URL)?

**Comment by ana** [comment]
https://lava.collabora.co.uk/scheduler/job/1178574

**Comment by gctucker** [comment]
Thanks!

Direct link to the v4l2 results: https://lava.collabora.co.uk/results/1178574/0_v4l2

PS: We'll need to review how this is stored in the backend, how regressions are found, how reports are generated etc...

**Comment by gctucker** [comment]
@ana There's an issue with the login prompt, see this job failure:
https://lava.collabora.co.uk/scheduler/job/1178670

Adding the `/ #` prompt solves the issue:
https://lava.collabora.co.uk/scheduler/job/1178718

So it looks like `lava-v2-jobs-from-api.py` should set this prompt with the Debian rootfs like for nfs boots.

**Comment by gctucker** [comment]
(job above was coming from a regular run on staging)

**Pull request merged**